python/lsst/pipe/analysis/plotUtils.py

```diff
@@ -795,3 +797,30 @@ def determineUberCalLabel(repoInfo, patch, coaddName="deep"):
         uberCalLabel = "None"
 
     return uberCalLabel
+
+def plotRhoStats(axes, rhoStats, filename):
+    for rhoIndex in range(1,6):
+        ## The fields to plot are best extracted by writing to and reading from disk
+        ## Create a relevant filename, although only temporary, to avoid clashes
+        rhoStats_filename = filename.replace('.png','_{}.asc'.format(rhoIndex))
+        rhoStats[rhoIndex].write(rhoStats_filename)
```

Comment:
Why are you doing this?

Reply:
Seems redundant but there's no easy way of extracting the required fields without writing on the disk first. Those fields are calculated only within the write method (!) in treecorr and I didn't want to mess with it.